src/components/_legal/FilesComponent/FilesComponent.tsx

```diff
@@ -0,0 +1,26 @@
+import { formatBytes } from '@/utils/format-bytes';
+import styles from './FilesComponent.module.scss';
+import type { Props } from './FilesComponent.types';
+
+const FilesComponent = ({ data }: Props) => {
+  return (
+    <section className={styles['FilesComponent']}>
+      {data.map(({ asset: { originalFilename, size, url } }, index) => (
+        <div
+          className={styles.linkWrapper}
+          key={index}
+        >
+          <a
+            href={`${url}?dl=${originalFilename}`}
```

Comment:
What is the `?dl` used for?

Reply:
It's a sanity feature that lets you download a file without leaving the site: https://www.sanity.io/docs/file-type